pkg/provider/gcp/secretmanager/client.go

```diff
@@ -378,6 +382,61 @@ func (c *Client) GetSecret(ctx context.Context, ref esv1beta1.ExternalSecretData
 	return []byte(val.String()), nil
 }
 
+func (c *Client) getSecretMetadata(ctx context.Context, ref esv1beta1.ExternalSecretDataRemoteRef) ([]byte, error) {
+	secret, err := c.smClient.GetSecret(ctx, &secretmanagerpb.GetSecretRequest{
+		Name: fmt.Sprintf("projects/%s/secrets/%s", c.store.ProjectID, ref.Key),
+	})
+
+	err = parseError(err)
+	if err != nil {
+		return nil, fmt.Errorf(errClientGetSecretAccess, err)
+	}
+
+	const (
+		annotations = "annotations/"
+		labels      = "labels/"
```

Comment:
Hey, sorry i gave some bad advice regarding the annotations/labels prefix, see here: https://github.com/external-secrets/external-secrets/issues/1874#issuecomment-1464993781. Let's do it as you have suggested initially :bow:

I noticed my fault while reviewing https://github.com/external-secrets/external-secrets/pull/2106.
Let's try to align the implementations of `metadataPolicy=Fetch` with the other in-flight PRs.

Reply:
Thank you for your review! Sure, let me fix the PR that way 🙂

Reply:
I've fixed it so would you take a look at it again when you have time?